aem/build.gradle.kts

```diff
@@ -1,10 +1,24 @@
 import com.cognifide.gradle.aem.common.ProgressLogger
 
 plugins {
-    id("com.cognifide.aem.instance")
+    id("com.cognifide.aem.environment")
 }
 
 aem {
+    docker {
+        healthCheck("http://example.com/en-us.html") {
+            status = 200
+            text = "English"
+        }
+        healthCheck("http://demo.example.com/en-us.html") {
```

Comment:
could you provide a link to that healthCheck impl? I think that better if such check will be introduced / using built-in GAP internal http client `aem.http { get("http://demo.example.com/en-us.html") { check(200, "English") } }`

Reply:
so that those health checks could be just a lambdas in which 

`healthCheck { aem.http { get("http://demo.example.com/en-us.html") { checkHtml(200, "English") } }`

etc

Reply:
Code is here :)
https://github.com/mierzwid/gradle-aem-plugin/pull/2/files